docs/design/autoware-architecture/index.md

```diff
@@ -0,0 +1,43 @@
+# Architecture Overview
+
+This page describes the architecture of Autoware.
+
+## Introduction
+
+The current Autoware is defined to be a layered architecture that clarifies each module's role and simplifies the interface between them. By doing so:
+
+- Autoware's internal processing becomes more transparent
+- Collaborative development is made easier because of the reduced interdependency between modules
+- Users can easily replace an existing module (e.g. localization) with their own software component by simply wrapping their software to fit in with Autoware's interface
+
+Note that the initial focus of this architecture design was solely on driving capability, and so the following features were left as future work:
+
+- Fail safe
+- HMI
+- Real-time processing
+- Redundant system
+- State monitoring system
+
+## High-level Architecture Design
+
+![Overview](image/AutowareArchitectureOverview.drawio.svg)
+
+This architecture consists of the following six stacks. Each of these design pages contains a more detailed set of requirements and use cases specific to that stack:
+
+- [Sensing Design](Sensing/index.md)
+- [Localization Design](Localization/index.md)
+- [Perception Design](Perception/index.md)
+- [Planning Design](Planning/index.md)
+- [Control Design](Control/index.md)
+- [VehicleInterface Design](Vehicle/index.md)
+- [Map Design](Map/index.md)
```

Comment:
[style] Use sentence case, and lower case for directories.

```suggestion
- [Sensing design](sensing/index.md)
- [Localization design](localization/index.md)
- [Perception design](perception/index.md)
- [Planning design](planning/index.md)
- [Control design](control/index.md)
- [VehicleInterface design](vehicle/index.md)
- [Map design](map/index.md)
```

Reply:
upps...orz sorry

Reply:
done in [f2bda3a](https://github.com/autowarefoundation/autoware-documentation/pull/86/commits/f2bda3aa760dbd6625e00c61e8a35884cfcd48e4)